### ISSUE TITLE
Receiving an error when trying to initialize player

### ISSUE THREAD

**Issue by michaeldcanady** [mental32/spotify.py#64]
When I run this code:
```
token = "user account token"
client_id = 'some client'
secret = 'some secret'

client = spotify.Client(client_id,secret)

async def main():
    user = await client.user_from_token(token)
    player = await user.get_player()


asyncio.get_event_loop().run_until_complete(main())
```

I receive the following error:

```
Warning (from warnings module):
  File "C:\Users\micha\AppData\Local\Programs\Python\Python38-32\lib\site-packages\spotify\client.py", line 164
    return await User.from_token(self, token)
RuntimeWarning: coroutine 'WrappedUser.__await__' was never awaited

Warning (from warnings module):
  File "C:\Users\micha\AppData\Local\Programs\Python\Python38-32\lib\site-packages\spotify\client.py", line 164
    return await User.from_token(self, token)
RuntimeWarning: coroutine 'User.from_token.<locals>.constructor' was never awaited
Traceback (most recent call last):
  File "E:/Projects/assistant/test.py", line 16, in <module>
    asyncio.get_event_loop().run_until_complete(main())
  File "C:\Users\micha\AppData\Local\Programs\Python\Python38-32\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "E:/Projects/assistant/test.py", line 12, in main
    user = await client.user_from_token(token)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python38-32\lib\site-packages\spotify\client.py", line 164, in user_from_token
    return await User.from_token(self, token)
TypeError: __await__() returned a coroutine
```

Python: 3.8.5
Spotify: spotify-0.10.2

**Comment by michaeldcanady** [comment]
I installed it by pip using "py -m pip install spotify"

**Comment by mental32** [comment]
Looks like a duplicate of #63 and will be solved by #68
